scylla_elasticsearch_integration/insert_data_into_scylla+elastic.py

```diff
@@ -5,97 +5,91 @@
 ### Using elasticsearch-py ###
 import csv
 from cassandra.cluster import Cluster
+from cassandra import ConsistencyLevel
 from elasticsearch import Elasticsearch
 
 import random
 import argparse
-import concurrent.futures
-from cassandra import ConsistencyLevel
-from cassandra.concurrent import execute_concurrent_with_args
-
 
 ## Script args and Help
 parser = argparse.ArgumentParser(add_help=True)
 
-parser.add_argument('-s', action="store", dest="SCYLLA_IP", default="127.0.0.1")
-parser.add_argument('-e', action="store", dest="ES_IP", default="127.0.0.1")
+parser.add_argument('-s', action="store", dest="SCYLLA_IP", default="localhost")
+parser.add_argument('-e', action="store", dest="ES_IP", default="http://localhost:9200")
 
 opts = parser.parse_args()
 
 SCYLLA_IP = opts.SCYLLA_IP.split(',')
 ES_IP = opts.ES_IP.split(',')
 
-
-## Define KS + Table
+## Define KeySpace + Table
 create_ks = "CREATE KEYSPACE IF NOT EXISTS catalog WITH replication = {'class' : 'NetworkTopologyStrategy', 'replication_factor' : 3};"
-create_t1 = "CREATE TABLE IF NOT EXISTS catalog.apparel (sku text, brand text, group text, sub_group text, color text, size text, gender text, PRIMARY KEY ((sku),color,size));"
-
+create_t1 = "CREATE TABLE IF NOT EXISTS catalog.apparel (sku text, brand text, group text, sub_group text, color text, size text, gender text, PRIMARY KEY ((sku), color, size));"
 
-## Loading the data
+# Loading the data
 def load_data(filename):
     data = []
     headers = []
-    with open(filename, "r") as f:
+    with open(filename, 'r') as f:
         reader = csv.reader(f)
         headers = next(reader) # read the headers line
-
-        for l in reader:
+        
```

Comment:
There are multiple occurrences in the code of empty lines but with tabs, is this related to any python formatter?

Afaik not, but the I think it would be good to give a formatter a pass in the code anyways, not necessarily in this PR.